common-lib/src/main/scala/com/gu/mediaservice/lib/cleanup/SupplierProcessors.scala

```diff
@@ -315,42 +315,17 @@ trait GettyProcessor {
 }
 
 object GettyXmpParser extends ImageProcessor with GettyProcessor {
-  def getSuppliersReference(image: Image)={
+  def getSuppliersReference(image: Image): Option[String] = {
     // String defined in ImageMetadataConvertor
     image.fileMetadata.getty.get("Asset ID").orElse(image.metadata.suppliersReference)
   }
 
   def apply(image: Image): Image = {
-    val excludedCredit = List(
-      "Replay Images", "newspix international", "i-images", "photoshot", "Ian Jones", "Photo News/Panoramic",
-      "Panoramic/Avalon", "Panoramic", "Avalon", "INS News Agency Ltd", "Discovery.", "EPA", "EMPICS", "Empics News",
-      "S&G and Barratts/EMPICS Sport", "EMPICS Sport", "EMPICS SPORT", "EMPICS Sports Photo Agency",
-      "Empics Sports Photography Ltd.", "EMPICS Entertainment", "Empics Entertainment", "MatchDay Images Limited",
-      "S&G and Barratts/EMPICS Archive", "PPAUK", "SWNS.COM", "Euan Cherry", "Plumb Images", "Mercury Press", "SWNS",
-      "Athena Pictures", "Flick.digital", "Matthew Horwood", "Focus Images Ltd", "www.scottishphotographer.com",
-      "ZUMAPRESS.com", "Huw Evans Agency", "Media6", "Alpha Press", "LNP", "Story Picture Agency"
-    )
-
-    val excludedSource = List(
-      "www.capitalpictures.com", "Replay Images", "UKTV", "PinPep", "Pinnacle Photo Agency Ltd", "News Images",
-      "London News Pictures Ltd", "Showtime", "Propaganda", "Equinox Features", "Athena Picture Agency Ltd",
-      "www.edinburghelitemedia.co.uk", "WALES NEWS SERVICE", "Sports Inc", "UK Sports Pics Ltd", "Blitz Pictures",
-      "Consolidated News Photos", "MI News & Sport Ltd", "Parsons Media", "Tom Nicholson", "SONY BMG MUSIC ENTERTAINMENT"
-    )
-
-    val isExcludedByCredit = image.metadata.credit.exists(isExcluded(_, excludedCredit))
-    val isExcludedBySource = image.metadata.source.exists(isExcluded(_, excludedSource))
-    val hasGettyMetadata = image.fileMetadata.getty.nonEmpty
 
+    val hasGettyMetadata = image.fileMetadata.getty.contains("Asset ID")
```

Comment:
@paperboyo didn't you say there's two spellings of `Asset ID`??

Reply:
Yes, but this (stupid!) copy of `xmp.GettyImagesGIFT` into our own `getty` contains [reconciliation](https://github.com/guardian/grid/blob/d2673a49e4d756ebefc619504cef9ef568bd27e4/image-loader/app/lib/imaging/FileMetadataReader.scala#L156). This is the *only* reason I’m not advocating removing this stupid copy altogether.